common/build/eslint-config-fluid/printed-configs/default.json

```diff
@@ -357,9 +384,24 @@
         "dot-notation": [
             "off"
         ],
-        "eol-last": [
+        "editorconfig/charset": [
+            "error"
+        ],
+        "editorconfig/eol-last": [
```

Comment:
I see some of the rules above that were previously enforced by typescript-eslint rules are now enforced here - is there a reason for that switch?

Reply:
I may not understand your question. But if you're referring to formatting rules, yes, I deliberately switched to editorconfig for the things that it checks because:

1. Typescript-eslint has several outstanding issues with their formatting rules. In some issue comments, they don't seem motivated to prioritize them since there are other well-adopted tools like prettier that nail the formatting task. This makes me avoid their formatting rules.
1. It's well-established in the repo. I.e. editorconfig files are part of our unofficial 'project template'.